docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,35 @@
 Version history
 ===============
 
+5.3.4 (2017 February 21)
+------------------------
+
+Bug fixes:
+
+* ImageJ
+   - fix for a NullPointerException when exporting images that were not opened via 
+     the Bio-Formats importer, and thus do not have a complete OMEXMLMetadata store.
+
+* Java 1.9
+   - fix compile and runtime errors to enable building with Java 1.9
+
+* ECAT7
+   - update to add support for different versions of ECAT7 files (thanks to Torsten Stöter)
+
+Updated build system:
+
+* updated dependency for `ome-model <https://github.com/ome/ome-model>`_ in the POM to 
+  version 5.4.0. This allows for improved ROI handling by enabling support for Shape 
+  objects with Transform attributes. OME-XML schema version remains unchanged as 
+  :model_doc:`OME schema 2016-06 </schemas/june-2016-2.html>`
```

Comment:
You also don't actually need the `/` in front of 'schemas/june-2016-2.html' but the link seems to be working anyway